### PR TITLE
Nextcloud blue upload bar + Notifications

### PR DESCRIPTION
**Pull request by williambargentball**
- Styles the upload bar in the nextcloud blue.
- Changes notification background colour to white.
- Adds faint border around file previews.
- Removes border radius form dropdown navigation (Looks cleaner)

Fix for #35 @mar1u5 @nextcloud/designers

### CODE REVIEW

File: apps/files/css/files.css [hunk 1]
```diff
@@ -293,6 +293,7 @@ table td.filename .thumbnail {
 	float: left;
 	position: absolute;
 	z-index: 4;
+        border: 1px solid #f7f7f7;
```
Comment (vincchan) [inline apps/files/css/files.css]: please fix the indentation

File: core/css/styles.css [hunk 2]
```diff
@@ -689,6 +689,7 @@ td.avatar {
 	position: absolute;
 	right: 0;
 	top: 0;
+        margin-top: 2px;
```
Comment (vincchan) [inline core/css/styles.css]: indentation here seems to be wrong as well

**Comment by vincchan** [comment]
@williambargent thanks for your contribution :) As a side note since you are a member of the nextcloud organisation you are able to create branches directly under the server repo. This enables reviewers to easily checkout your branch and your changes.

File: core/css/jquery-ui-fixes.css [hunk 3]
```diff
@@ -17,8 +17,8 @@
 }
 .ui-widget-header {
 	border: 1px solid #0082c9;
-	background: #0082c9 url('images/ui-bg_flat_35_1d2d44_40x100.png') 50% 50% repeat-x;
-	color: #ffffff;
+	background: #0082c9;
+	color: #0082c9;
```
Comment (jancborchardt) [inline core/css/jquery-ui-fixes.css]: The background and text color is the same here. Is that intentional?

**Comment by jancborchardt** [comment]
Hey, so these are quite a lot of unrelated changes for one pull request. I’d say it would be good to separate these since one fixes an issue, one is a good enhancement, and the other two I would rather not change. :) Detailed comments:

> - Styles the upload bar in the nextcloud blue

Great! That fixes #35 

> - Changes notification background colour to white.

That’s a actually a good enhancement to make the look nicer! For the future it would be good to separate this change out because it has nothing to do with the other things.

> - Adds faint border around file previews.

What’s the reasoning behind this change?

> - Removes border radius form dropdown navigation (Looks cleaner)

The rounded corner looks cleaner and friendlier than a sharp corner. I can recommend reading http://www.uiandus.com/blog/2009/7/27/realizations-of-rounded-rectangles.html :)

**Comment by williambargent** [comment]
screenshot 2

The file table, navigation and previews are all line and corners, I felt it looked nicer rather than suddenly jumping to curves.

**Comment by jancborchardt** [comment]
@williambargent no worries. :) The reason there are no borders around the icon is that it’s unneeded visual noise. And the reason for the border-radius on the bottom of the menu dropdowns is that it’s a tiny detail fitting with the other round corners in the interface (logo, round avatars, all icons such as filetypes).

**Pull request closed without merging**